### ISSUE TITLE
Comments tab disabled depending on browser zoom

### ISSUE THREAD

**Issue by arielleandrews** [bcgov/foi-flow#2580]
**Describe the bug in current situation**
Certain zoom settings on the user's browser make the user unable to access the Comments tab on a request. A user in the production environment was not able to click the Comments tab with their browser zoom set at 100%.

**Link bug to the User Story**

**Impact of this bug**
Impact is high. Bug makes it so a user cannot access the Comments tab on a request and are therefore unable to use the comments feature.

**Chance of Occurring (high/medium/low/very low)**
Medium. Does not appear to be occurring with many users at this time, may depend on screen resolution etc. 

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**
Occurs in Production environment.

**Steps to Reproduce**
1. Go to the request details page on any request
2. See if Comments tab is clickable
3. Change zoom settings (in or out)
4. See if Comments tab is clickable

**Actual/ observed behaviour/ results**
Users should be able to access the Comments tab at any zoom setting

**Expected behaviour**
A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by abin-aot** [comment]
This bug has been fixed on different resolutions, with various zoom levels. The root cause of the issue is like on certain Request states like CFR and above, the "Due days " panel which has the due day count was overlapping the Tab/Tab Headers - this made certain users unable to click the comment tab.. I fixed this issue removing extra padding on lower resolutons and zoom levels. But there futuristic issue which we are foreseeing like , when the new tabs comes as part of CFR Form, applicant correspondence etc. this issue can happen again since there is less real estate to accommodate. One possible solution for this problem, is to rearrange left panel contents like Tabs, or remove the HOME icon and manage the real estate - this will be a quicker solution. Second solution, which is the best one, permanent one is to make entire web application FLUID and responsive to any possible resolution. This will solve any overlay issues and ripple affect of fixing one UX component to another. This (second) solution has been discussed earlier - refer #2159 - Its a story now , but need to scoped as an EPIC since effort might span across few sprints.  Note : Commented after a meeting with @lmullane  @m-prodan @JHarrietha-AOT @aparna-aot
